modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -43,10 +43,12 @@ run.meta.analysis.pft <- function(pft, iterations, random = TRUE, threshold = 1.
   PEcAn.logger::logger.info("-------------------------------------------------------------------")
   
   ## Load trait data for PFT
-  load(file.path(pft$outdir, "trait.data.Rdata"))
-  load(file.path(pft$outdir, "prior.distns.Rdata"))
+  trait_env <- new.env()
```

Comment:
To load the file contents into their own object rather than the function's environment; this is (1) the recommended practice to avoid overwriting any existing objects with the same name as the loaded object(s), and (2) avoids "no visible binding for global variable" warnings from R CMD check.

Reply:
@dlebauer I now see you were asking for an explanatory comment in the code, not just in this thread. In the spirit of "make the code so clear it doesn't need comments", would better variable names have helped you follow what was happening here?

Reply:
For cases where other file formats would actually be preferable, I support this change. But I see `load`'s "assign in the parent env by default" behavior as a minor historical annoyance rather than a reason to avoid the function entirely: .Rdata is a pretty good file format that's well suited for many applications, `load` is the obvious way to read it, and at this point even if we changed the format for new files we'd probably want to keep supporting .Rdata indefinitely so that old meta-analysis results can be read and updated.